DemoServer/client/src/store/actions/tracking.ts

```diff
@@ -56,29 +66,20 @@ export function initTrackingData(gtmContainerId: string): DemoThunkAction {
         }
 
         dispatch(saveGtmContainerId(gtmContainerId));
-
-        const consentStatus = CookieJar.getConsentStatus();
```

Comment:
Please remove the `CookieJar.getConsentStatus()` function and all related types if it's not used.

Reply:
Done.

Reply:
Have you checked if the `ConstentStatus` type is used anywhere?

Reply:
Removed the unused type.